x-pack/plugins/infra/public/pages/logs/settings/index_pattern_selector.tsx

```diff
@@ -83,6 +83,6 @@ export const IndexPatternSelector: React.FC<{
 };
 
 const indexPatternSelectorPlaceholder = i18n.translate(
```

Comment:
This is tricky but should we rename these variables as well?

I made some changes recently to remove IndexPattern types that were recently deprecated but I didn't rename the variables because there were just so many occurrences so I don't know how we handle this.

Reply:
It's a good question. I tried (for the most part) to leave variables etc alone. They definitely need updating but I think it'd make more sense to make a followup issue and do it all in one go. Reason being there's quite a lot of changes to be made in the server side code (especially where we have the "resolver" code). That way we can tick off our box on https://github.com/elastic/kibana/issues/100844 ASAP.